plasmapy/formulary/parameters.py

```diff
@@ -897,6 +925,10 @@ def gyrofrequency(B: u.T, particle="e-", signed=False, Z=None) -> u.rad / u.s:
     return omega_ci
 
 
+oc = wc = gyrofrequency
+""" Alias to :func:`gyrofrequency`. """
```

Comment:
Looking at the doc build, it seems like we're going to have to duplicate these two lines for `oc` and `wc` both. 
https://plasmapy--865.org.readthedocs.build/en/865/formulary/parameters.html

Reply:
Yeah, I'm guessing for the documentation to show up correctly they would have to be defined in independent statements.